soak-tests-common/src/main/java/com/hazelcast/jet/tests/common/AbstractSoakTest.java

```diff
@@ -282,4 +282,14 @@ protected static void assertFalse(String message, boolean actual) {
             throw new AssertionError(message);
         }
     }
+
+    protected static void assertNotNull(Object actual) {
```

Comment:
Nit : You don't need string concatenation of "actual" parameter. It is null. Just make the message
assertNotNull("expected: not null, actual: null" , ...)

Reply:
Good point, I just copy-pasted the method from other assert method and haven't think about it more. I will adjust it.